### PR TITLE
Adding support for latest JsInterop syntax

### LINKED ISSUE #66: Update to GWT 2.8 new syntax : JS_RC
The GWT team is preparing the 2.8 release, and we should update to use the RC once it’s out.

### PR DESCRIPTION
**Pull request by manolo**
Connected with vaadin/gwt-api-generator#54
Fixes #66 

[Review on Reviewable](https://reviewable.io/reviews/vaadin/gwt-polymer-elements/75)

### CODE REVIEW

**Comment by frami** [comment]
I need the latest JsInterop syntax. When will a gwt-polymer-elements release with the latest JsInterop syntax available?

**Comment by cpboyd** [comment]
Review status: 0 of 5 files reviewed at latest revision, 1 unresolved discussion.

---

_[package.json, line 11 [r1]](https://reviewable.io:443/reviews/vaadin/gwt-polymer-elements/75#-K7Mr9RyltqND1XN8xfS:-K7Mr9RyltqND1XN8xfT:1766216213) ([raw file](https://github.com/vaadin/gwt-polymer-elements/blob/0b876d8de19873fa029840a17cc412d61c1022d8/package.json#L11)):_
Does this branch of vaadin-grid exist publicly?

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/gwt-polymer-elements/75)_

**Comment by cpboyd** [comment]
Edit: JsInterop reference was causing failure in Maven build with GWT 2.8.0-SNAPSHOT on January 6.
The build now works fine again, but indicates that the SNAPSHOT is still in an unstable phase.

Likely related to gwtproject/gwt#9235

File: package.json [hunk 1]
```diff
@@ -5,13 +5,13 @@
   "author": "Vaadin Ltd",
   "license": "Apache License 2.0",
   "dependencies": {
-    "gwt-api-generator": "vaadin/gwt-api-generator#1.2.1"
+    "gwt-api-generator": "vaadin/gwt-api-generator#JS_RC"
   },
   "scripts": {
-    "postinstall":"node node_modules/gwt-api-generator/bin/gwt-api-generator.js --package=PolymerElements/paper-elements,PolymerElements/iron-elements,vaadin/vaadin-grid#fa92309  --artifactId=vaadin-gwt-polymer-elements --groupId=com.vaadin.polymer --pom && mvn clean install javadoc:jar"
+    "postinstall":"node node_modules/gwt-api-generator/bin/gwt-api-generator.js --package=PolymerElements/paper-elements,PolymerElements/iron-elements,vaadin/vaadin-grid#feature/apidocs  --artifactId=vaadin-gwt-polymer-elements --groupId=com.vaadin.polymer --pom && mvn clean install javadoc:jar"
```
Comment (tomivirkki) [inline package.json]: branch doesn't exist vaadin/vaadin-grid#feature/apidocs

**Comment by manolo** [comment]
Review status: 0 of 12 files reviewed at latest revision, 2 unresolved discussions.

---

_[package.json, line 11 [r1]](https://reviewable.io:443/reviews/vaadin/gwt-polymer-elements/75#-K3E6jLB2XSEXAur3x5V-r1-11:-K7z4mtlfgkMx8ZjOlfQ:1527665442) ([raw file](https://github.com/vaadin/gwt-polymer-elements/blob/0b876d8de19873fa029840a17cc412d61c1022d8/package.json#L11)):_
it was needed when I did the PR, the branch is merged so changed to #master

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/gwt-polymer-elements/75)_

**Comment by manolo** [comment]
Review status: 0 of 12 files reviewed at latest revision, 2 unresolved discussions.

---

_[package.json, line 11 [r1]](https://reviewable.io:443/reviews/vaadin/gwt-polymer-elements/75#-K7Mr9RyltqND1XN8xfS:-K7z4zmW2hNFNOoPEKnD:-1972584617) ([raw file](https://github.com/vaadin/gwt-polymer-elements/blob/0b876d8de19873fa029840a17cc412d61c1022d8/package.json#L11)):_
It is in master

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/gwt-polymer-elements/75)_

**Comment by manolo** [comment]
@frami : use gwt-polymer-elements-1.2.3.0-SNAPSHOT and gwt-2.8.0-SNAPSHOT in your project and should work

---

Review status: 0 of 12 files reviewed at latest revision, 2 unresolved discussions.

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/gwt-polymer-elements/75#-:-K7z54_XxZ5ZkHlk-aNz:-1310729817)_

**Comment by frami** [comment]
Thank you, the snapshot version works for me.

**Comment by tehapo** [comment]
LGTM now that changes discussed in Skype are resolved. :+1: 

---

Reviewed 2 of 5 files at r1, 7 of 10 files at r2, 8 of 13 files at r3, 5 of 5 files at r4.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/gwt-polymer-elements/75#-:-K8YlsOH2aZPvIwKorDR:1446619985)_

**Comment by tomivirkki** [comment]
_[package.json, line 8 [r6]](https://reviewable.io:443/reviews/vaadin/gwt-polymer-elements/75#-K8Z9mrh6rfLq39z_RBk:-K8Z9mrh6rfLq39z_RBl:-1608420510) ([raw file](https://github.com/vaadin/gwt-polymer-elements/blob/b4d1bd5bef3e189844009836c8fecab39b080d0a/package.json#L8)):_
This JS_RC branch is going to be deleted once https://github.com/vaadin/gwt-api-generator/pull/54 gets merged

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/gwt-polymer-elements/75)_

**Comment by manolo** [comment]
Fixes #83 
Fixes #84 
Fixes #86 
Fixes #81 
Fixes #79 
Fixes #49 
Fixes #48

**Comment by tehapo** [comment]
Reviewed 2 of 2 files at r5, 1 of 1 files at r6, 3 of 3 files at r7.
Review status: 22 of 23 files reviewed at latest revision, 2 unresolved discussions.

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/gwt-polymer-elements/75)_

**Pull request merged**